src/tzdata/build.jl

```diff
@@ -38,25 +38,53 @@ function build(
         end
     end
 
-    archive = joinpath(archive_dir, "tzdata$version.tar.gz")
-
-    # Avoid downloading a tzdata archive if we already have a local copy
-    if version == "latest" || !isfile(archive)
-        @info "Downloading $version tzdata"
-        archive = tzdata_download(version, archive_dir)
-
+    if VERSION >= v"1.4"
+        now_utc = now(Dates.UTC)
+        # todo: I don't have latest here and it should not be used as latest
         if version == "latest"
-            m = match(TZDATA_VERSION_REGEX, basename(archive))
+            m = match(TZDATA_VERSION_REGEX, "tzdata$version.tar.gz")
             if m !== nothing
                 version = m.match
                 @info "Latest tzdata is $version"
             end
+
+            artifact_dir = @artifact_str "tzdata_$version"
+            version = tzdata_version_dir(artifact_dir)
+            set_latest(version, now_utc)
+        end
+        artifact_dir = @artifact_str "tzdata_$version"
+        set_latest(version, now_utc)
+        #if !isfile(LATEST_FILE)
+        #    set_latest(version, now_utc)
+        #end
+
+        if !isempty(tz_source_dir)
+            @info "Copying region data from version $version"
+            for region in setdiff(regions, CUSTOM_REGIONS)
+                cp(joinpath(artifact_dir, region), joinpath(tz_source_dir, region), force=true)
```

Comment:
```suggestion
                cp(joinpath(artifact_dir, region), tz_source_dir, force=true)
```

Reply:
Could you elaborate more on why to do this change? With this change, the tests were crashing, and the package does not build.
I'm returning it back because it fails tests: https://travis-ci.com/github/JuliaTime/TimeZones.jl/jobs/361052429